sklearn/metrics/tests/test_ranking.py

```diff
@@ -601,9 +624,9 @@ def test_precision_recall_curve_toydata():
         assert_raises(Exception, average_precision_score, y_true, y_score,
                       average="weighted")
         assert_almost_equal(average_precision_score(y_true, y_score,
-                            average="samples"), 1.)
+                                                    average="samples"), 1.)
```

Comment:
please revert all these unrelated changes. It makes the review harder

Reply:
ok, I will update it. My IDE did that :/